tests/tasks_test.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+import pytest
+import responses
+
+from filestack import Transformation, AudioVisual
+from filestack import config
+
+APIKEY = 'SOMEAPIKEY'
+HANDLE = 'SOMEHANDLE'
+EXTERNAL_URL = 'SOMEEXTERNALURL'
+
+
+@pytest.fixture
+def transform():
+    return Transformation(apikey=APIKEY, external_url=EXTERNAL_URL)
+
+
+def test_createpdf(transform):
+    target_url = '{}/{}/pdfcreate/{}/[v7MSSKswR0mvEwZS9LD0,Sr5CrtQPSs5TTZzor1Cw]'.format(config.CDN_URL, APIKEY, EXTERNAL_URL)
```

Comment:
file handle should be 20 character long

Reply:
There would be list of handles (array of images) comma separated need to create PDF file and these handles have 20 characters long.